cmd/scollector/main.go

```diff
@@ -358,3 +327,176 @@ func printPut(c chan *opentsdb.DataPoint) {
 		slog.Info(string(b))
 	}
 }
+
+func toToml(fname string) {
```

Comment:
Is there an end of life plan for this? When can we delete this code? I assume new config directives won't need to be updated here?